game_eggs/steamcmd_servers/satisfactory/README.md

```diff
@@ -96,65 +108,84 @@ ___
 
 ___
+
 ### Server Initialization
+
 For a server to be fully "initialized", a client who owns the game must log into the server to "claim" it and create an administrator password. Then, a new session can be created via the "Create Game" tab in-game, or an existing save file can be uploaded (see [Save Files](#save-files) below).
 
 Misc. settings listed below can be configured by an admin client via the game's "Server Settings" tab, and are currently **not** set via the Egg:
+
 - Server Password
 - Admin Password
 - Auto-Save on Player Disconnect
 - Pause When No Players Online
 - ...and possibly more as the client's UI is developed further for more configuration options.
+
 ___
+
 ### Save Files
+
 An existing save file (including single-player saves) can currently be uploaded to the server via two different methods:
 - "Manage Saves" tab via a client in-game (Recommended)
 - Manually via the File Manager or SFTP
 
 Save files are located in this directory:
-```
+
+```md
 /home/container/.config/Epic/FactoryGame/Saved/SaveGames/server
 ```
+
 *Note: A manually uploaded save will only load if it is (a.) loaded manually via the "Manage Saves" tab in-game, (b.) it is the only save file present, or (c.) its existing session name (not its file name) matches the existing save's session name *and* has the most recent time stamp.*
 
 ***Warning:*** Stopping the server **does not** currently save your game! Ensure it is saved before stopping the server.
 
 If you have forgotten your administrator password or would generally like to reset your server as if it were new, you can delete the following file:
-```
+
+```md
 /home/container/.config/Epic/FactoryGame/Saved/SaveGames/ServerSettings.<your_server_query_port>
 ```
+
 ___
+
 ### Console Commands
+
 As of v0.5.1.2, the console tab in the client server manager is the only way to execute commands. Entering commands via Pterodactyl do nothing.
 
 [List of known commands can be found via the Wiki.](https://satisfactory.fandom.com/wiki/Dedicated_servers#Console_commands)
 ___
+
 ### Errors/Warnings
+
 The following errors or warnings you see in the console can safely be ignored:
 
 ```log
 steamclient.so: cannot open shared object file: No such file or directory
 [S_API] SteamAPI_Init(): Loaded '/home/container/.steam/sdk64/steamclient.so' OK.  (First tried local 'steamclient.so')
 LogSteamShared: Warning: Steam Dedicated Server API failed to initialize.
 ```
-<sup>↑ The local file of 'steamclient.so' was attempted to be loaded, but could not because it is not present, causing the warning message. However, the backup `/home/container/.steam/sdk64/steamclient.so` is loaded successfully (this is the correct behavior according to the [Wiki](https://satisfactory.fandom.com/wiki/Dedicated_servers#SteamAPI_Init.28.29:_Sys_LoadModule_filed_to_load:_.2Fpath.2Fto.2F.steam.2Fsdk64.2Fsteamclient.so)).</sup>
```

Comment:
I personally think it's easier to read this section with these comments as "footnotes" by using the `<sup>` tag, though stylistic choices are subjective.

Reply:
Yea, I can lean either way, but I try to avoid inline HTML in a markdown file.